example/lambdas.yml

```diff
@@ -4,8 +4,8 @@ CNMToCMA:
   runtime: java8
   memory: 128
   s3Source:
-    bucket: '{{buckets.shared.name}}'
-    key: daacs/podaac/cnmToGranule-1539697831000.zip
+    bucket: '{{buckets.internal.name}}'
```

Comment:
This needs both a PR and a deployment solution.   For *now* leaving it in the location I can publish it.

Reply:
This needs to move back to the shared bucket before merging.

In the meantime we can PR to PODAAC and see if they will put it somewhere we can get to.

Reply:
I'll update it and have mark boyd publish my current version.  I'm working on getting access to the PODAAC repo